src/v/redpanda/application.cc

```diff
@@ -560,6 +560,7 @@ manager_config_from_global_config(scheduling_groups& sgs) {
       config::shard_local_cfg().log_segment_size.bind(),
```

Comment:
nit: Do we really need to backport these commits to backport timequeries?

Reply:
It's necessary for the tests: without these, the tests are subject to log size jitter and can't reliably satisfy their success conditions.